sysfs/pci_device.go

```diff
@@ -175,7 +205,7 @@ func (fs FS) parsePciDevice(name string) (*PciDevice, error) {
 	}
 
 	// These files must exist in a device directory.
-	for _, f := range [...]string{"class", "vendor", "device", "subsystem_vendor", "subsystem_device", "revision"} {
+	for _, f := range [...]string{"class", "vendor", "device", "subsystem_vendor", "subsystem_device", "revision", "numa_node"} {
```

Comment:
Are we sure `numa_node` is always going to exist? Even on older kernels? We tend to support some pretty old kernel versions here.